docs/docs/migration-guide.mdx

```diff
@@ -1225,6 +1225,24 @@ confidences. This change can affect the maximum confidence predicted by the
 However, the previous behavior can still be enforced by setting
 `ranking_length=10` and `renormalize_confidences=True`.
 
+#### Removed Policies
+
+Several dialogue policies that were deprecated in Rasa 2.x have been removed in Rasa 3.x.
+If you are trying to migrate a config file with a removed policy,
+consult the migration guides for the individual policies:
+
+- `FallbackPolicy` [migration guide](#manually-migrating-from-the-fallback-policy)
+- `TwoStageFallbackPolicy` [migration guide](#manually-migrating-from-the-two-stage-fallback-policy)
+- `MappingPolicy` [migration guide](#manually-migrating-from-the-mapping-policy)
+- `FormPolicy` [migration guide](#forms)
+- `SklearnPolicy` should be replaced with [TEDPolicy](/policies#ted-policy)
```

Comment:
Is there any indication of which settings to use when migrating? Even an entry to the changelog would help since there is no other info on this

Reply:
You are referring to the SklearnPolicy specifically or all of the above policies?

Reply:
sklearn specifically, my bad

Reply:
@melindaloubser1 I can't find anything on SklearnPolicy other than [this changelog](https://rasa.com/docs/rasa/changelog#deprecations-and-removals-6) entry (see #6658). The issue linked in the change log entry doesn't say anything about why or how it should be replaced.

I think the SklearnPolicy can take arbitrary sklearn models as an argument so it will be hard to give very specific guidance. 

How does the following strike you:

> - `SklearnPolicy` should be replaced with [TEDPolicy](/policies#ted-policy). We recommend the [default TED config](https://rasa.com/docs/rasa/model-configuration#suggested-config) as a starting point.

Reply:
Gotcha, in that case yes sounds good

Reply:
Sweet!